azure-gaming-pr/reference-architectures/cognitive-text-to-speech.md

```diff
@@ -25,20 +25,22 @@ This article will describe the architecture used in [this sample on GitHub](http
 - [Azure Function](https://docs.microsoft.com/azure/azure-functions/functions-overview) - Simplest way to run code on-demand in the cloud.
 - [Azure Content Moderator](https://docs.microsoft.com/azure/cognitive-services/content-moderator/overview) - Included to avoid detect profanity or other undesirable language.
 - [Azure Text Analytics](https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview) - This service detects the language used by the player, which is required for the Azure Speech service. Alternatively, [Azure Content Moderator](https://docs.microsoft.com/azure/cognitive-services/content-moderator/overview) can also detect the language.
-- [Azure Speech](https://docs.microsoft.com/azure/cognitive-services/speech-service/text-to-speech) - The service that provides the text to speech functionality.
+- [Azure Speech](https://docs.microsoft.com/azure/cognitive-services/speech-service/text-to-speech) - This service provides the Text-to-Speech functionality. With Text-to-Speech, you can convert input text into humanlike synthesized speech by using [prebuilt neural voices](https://docs.microsoft.com/azure/cognitive-services/speech-service/language-support?tabs=speechtotext#prebuilt-neural-voices) or [custom neural voices](https://docs.microsoft.com/azure/cognitive-services/speech-service/custom-neural-voice). 
 - [Azure Premium Blob Storage](https://docs.microsoft.com/azure/storage/blobs/storage-blob-storage-tiers) - Selected due to latency requirements, as the standard Azure Blob Storage may have limitations if the voice-file read is on demand during game play.
 
 ## Architecture Considerations
 
-You only need to create a single Azure Event Hub namespace that contains the 2 Azure Event Hubs used for sending and receiving the messages respectively.
+For [Azure Speech](https://docs.microsoft.com/azure/cognitive-services/speech-service/text-to-speech), max number of transactions per second per Speech service resource is 200 transactions per second (TPS) for S0 (standard pricing tier) users. You can submit a request to increase this amount. For details, see [Text-to-speech quotas and limits per resource](https://docs.microsoft.com/azure/cognitive-services/speech-service/speech-services-quotas-and-limits#text-to-speech-quotas-and-limits-per-resource). You can also use the [Long Audio API](https://docs.microsoft.com/azure/cognitive-services/speech-service/long-audio-api) to asynchronously synthesize text-to-speech files longer than 10 minutes (for example, audio books or lectures). 
```

Comment:
if you have a game-related usage, add them to the example and then include audio books or lectures.

Reply:
Sorry for the confusion.  I've deleted the part in brackets to make it general.